app/views/static/about.html.haml

```diff
@@ -35,3 +35,8 @@
   = link_to "useiconic.com", "https://useiconic.com/"
   from
   #{link_to "the Noun Project", "http://thenounproject.com/"}.
+  
+  %p
+    PlanningAlerts receives free access to use Google Maps services through the&nbsp;
+    %a(href="https://www.google.com/grants/")> Google Grants Program
```

Comment:
This link should use the `link_to` Rails method, like the one on [line 35 above](https://github.com/openaustralia/planningalerts/pull/1107/files#diff-df7147916bb65cbad52262ae3a631dcfR35). See http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to.